src/index.js

```diff
@@ -17,21 +18,25 @@ const elementGenerator = (typeName, className, content, idName) => {
   return element;
 };
 
-const todoTasks = [
+// eslint-disable-next-line import/no-mutable-exports
```

Comment:
- [ ] Same here. I would suggest fixing this in code instead of `disabling linters`. All rules are applied for the purpose to write code in a best practice manner.

Reply:
It is done